server/util.go

```diff
@@ -197,9 +199,39 @@ func CreateSDKLogger(ctx *Context, out io.Writer) (log.Logger, error) {
 	// Check if the CometBFT flag for trace logging is set and enable stack traces if so.
 	opts = append(opts, log.TraceOption(ctx.Viper.GetBool("trace"))) // cmtcli.TraceFlag
 
+	// Error fields should be set under error object
+	zerolog.ErrorFieldName = "error"
```

Comment:
This is the default in the library so we don't need to do it.
https://pkg.go.dev/github.com/rs/zerolog#section-readme